File: Secao07 - Comandos de repetição/Exercicios em Java/Exercicio06.java
package br.com.udemy.secao07;

import java.util.Scanner;

public class Exercicio06 {

	public static void main(String[] args) {
		Scanner teclado = new Scanner(System.in);
		int num, valor;
		
		System.out.print("Informe um numero de 1 a 10: ");
		num = teclado.nextInt();
		
		while (num < 1 || num > 10) {
			System.out.println("Erro! Numero deve estar entre 1 e 10.");
			System.out.println(" ");
			System.out.print("Informe um numero de 1 a 10: ");
			num = teclado.nextInt();
		}
		System.out.println(" ");
		System.out.println("Tabuada do numero " + num);
		for(int i = 0; i <= 10; i++) {
			valor = num * i;
			System.out.println(num + " X " + i + " = " + valor);
		}
		teclado.close();
	}

}

File: Secao07 - Comandos de repetição/Exercicios em Python/Exercicio06.py
# -*- coding: utf-8 -*-
print("Tabuada de 1 a 10\n")
num1 = int(input("Informe numero de 1 a 10 para calculo da tabuada: "))

while num1 > 10 or num1 < 1:
    print("Numero deve estar entre 1 e 10.\n")
    num1 = int(input("Informe numero de 1 a 10 para calculo da tabuada: "))
print("Tabuada do numero {0}.".format(num1))

for n in range(1,11):
    result = num1 * n
    print("{0} X {1} = {2}".format(num1, n, result))